open-api/rest-catalog-open-api.yaml

```diff
@@ -134,9 +134,19 @@ paths:
     post:
       tags:
         - OAuth2 API
-      summary: Get a token using an OAuth2 flow
+      summary: Get a token using an OAuth2 flow (DEPRECATED for REMOVAL)
```

Comment:
We should add the field  `deprecated: true` to mark it as deprecated.

 ```
 /v1/oauth/tokens:

    post:
      tags:
        - OAuth2 API
      summary: Get a token using an OAuth2 flow
      deprecated: true
      operationId: getToken
      description: ---
```

Reply:
more info: https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#fixed-fields-8